### ISSUE TITLE
I can't find the token to log into Jupyter Lab

### ISSUE THREAD

**Issue by khang-1111** [ValyrianTech/ComfyUI_with_Flux#5]
Usually, when I create a new pod on Runpod with an RTX 3090, I just open the log, type 'token,' and it appears. However, today I created a new pod with the V100 SXM2 card and can't find any token in the log (I tried those keywords like Token, Server, Lab, etc.) . I have restarted the pod, stopped it, and started it again, but there is still no token in the log so I can't access Jupyter Lab to download some Upscale models.

![2024-10-27 23_05_34-My Pods](https://github.com/user-attachments/assets/115233ff-f0a1-4096-8166-4ed592acecfa)

![2024-10-27 23_06_04-My Pods](https://github.com/user-attachments/assets/b6156cbb-3415-45c8-a9bc-7ee46d32b920)

**Comment by WouterGlorieux** [comment]
Yes, I noticed this too, unfortunately it is nothing that I can change. I think Runpod made some changes to how the container logs work, I suspect they changed the logging level or something and there are a whole bunch of messages no longer recorded in the logs, not just the token for JupyterLab, but other programs too, and other templates.

I already notified Runpod support today, waiting for a response.

Until then, you can start a web terminal and use the command 'jupyter server list' to get the token to log into JupyterLab.

**Comment by khang-1111** [comment]
> Yes, I noticed this too, unfortunately it is nothing that I can change. I think Runpod made some changes to how the container logs work, I suspect they changed the logging level or something and there are a whole bunch of messages no longer recorded in the logs, not just the token for JupyterLab, but other programs too, and other templates.
> 
> I already notified Runpod support today, waiting for a response.
> 
> Until then, you can start a web terminal and use the command 'jupyter server list' to get the token to log into JupyterLab.

Thank you very much I can access JupyterLab now.

**Comment by Eleksar387** [comment]
I have the same issue but if I use the token from the 'jupyter server list' command I get 'Invalid credentials'

**Comment by WouterGlorieux** [comment]
@Eleksar387 I'm not sure what is causing your problem, make sure you got the correct token copied, maybe you missed a letter or something. Also I think it is better to use right-click and copy than use shortcut keys, I've had sometimes that I thought I copied the token but it didn't.

**Comment by Eleksar387** [comment]
@WouterGlorieux Whatever the reason was. I run a new pod and it worked! Love what you are doing! Thank you.

**Comment by letscreate321** [comment]
@WouterGlorieux  thank you so much! It worked!
